src/Ombi/ClientApp/src/app/settings/plex/components/plex-server-dialog/plex-server-dialog.component.scss

```
@@ -1,27 +1,439 @@
+        letter-spacing: 0.12em;
+        text-transform: uppercase;
+        color: $plex-text-faint;
+    }
+
+    &__title {
+        margin: 4px 0 4px;
+        font-size: 20px;
+        font-weight: 700;
+        letter-spacing: -0.01em;
+        color: #fff;
+        word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)